click_web/web_click_types.py

```diff
@@ -1,5 +1,8 @@
 """
 Module contains click types that could be useful on web pages and provide form validation by just setting type.
+Custom web click types need to be imported into the main script not the app.py that flask runs with
+    from click_web.web_click_types import EMAIL_TYPE
+and replacing email type with the custom click type desired
```

Comment:
First part is good and provides clarification, not so sure about the last part
> and replacing email type with the custom click type desired

Maybe a code example would be the clearest as now there is only one type in the module (EMAIL) and instructing users to replace it with other custom types seems confusing.
And as users of this library must understand basic click concepts in order to produce scripts they should be familiar with click types if they see an example.

```from click_web.web_click_types import EMAIL_TYPE
@click.command()
@click.option("--the_email", type=EMAIL_TYPE)
def email(the_email):
    click.echo(f"{the_email} is a valid email syntax.")
```